components/card/article/src/index.js

```diff
@@ -149,6 +152,7 @@ CardArticle.propTypes = {
 
 CardArticle.defaultProps = {
   featured: false,
+  tagClassName: null,
```

Comment:
I'd expect some string as default value regarding the prop types, if there's none we can just remove it.